### PR TITLE
1.5.0-beta01 CHANGELOG

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,113 @@
+# 1.5.0-beta01 (July 2023)
+
+## Common
+
+### Features
+- [Dialog in common](https://github.com/JetBrains/compose-multiplatform-core/pull/632)
+- [Popup in common](https://github.com/JetBrains/compose-multiplatform-core/pull/611)
```
Comment (MatkovIvan) [inline CHANGELOG.md:7]:
```suggestion
- [`Dialog` API is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/632)
- [`Popup` API is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/611)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,113 @@
...
+- [Insets in common](https://github.com/JetBrains/compose-multiplatform-core/pull/586)
+- [A warning if compose.kotlinCompilerPlugin is set to androidx.compose.compiler.compiler](https://github.com/JetBrains/compose-multiplatform/pull/3313)
```
Comment (MatkovIvan) [inline CHANGELOG.md:9]:
```suggestion
- [A warning if `compose.kotlinCompilerPlugin` is set to `androidx.compose.compiler.compiler`](https://github.com/JetBrains/compose-multiplatform/pull/3313)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,113 @@
+# 1.5.0-beta01 (July 2023)
+
+## Common
+
+### Features
+- [Dialog in common](https://github.com/JetBrains/compose-multiplatform-core/pull/632)
+- [Popup in common](https://github.com/JetBrains/compose-multiplatform-core/pull/611)
+- [Insets in common](https://github.com/JetBrains/compose-multiplatform-core/pull/586)
```
Comment (MatkovIvan) [inline CHANGELOG.md:8]:
```suggestion
- [`WindowInsets` API is available to use from common source set](https://github.com/JetBrains/compose-multiplatform-core/pull/586)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,113 @@
...
+### API Changes
+ - Resource management was reimplemented. Follow [the guide in the PR](https://github.com/JetBrains/compose-multiplatform/pull/3340) to support new feautures
+
+## Desktop
+
+### Features
+- [Swing interop. Experimental off-screen rendering on graphics](https://github.com/JetBrains/compose-multiplatform-core/pull/601)
+- [Swing interop. ComposePanel that can be disposed manually](https://github.com/JetBrains/compose-multiplatform-core/pull/620)
+- [Add semantic properties to DialogWindow, Popup and Dialog](https://github.com/JetBrains/compose-multiplatform-core/pull/698)
+- [Use Segoe UI on Windows by default](https://github.com/JetBrains/compose-multiplatform-core/pull/557)
```
Comment (MatkovIvan) [inline CHANGELOG.md:59]:
```suggestion
- [Use `Segoe UI` as sans serif font on Windows](https://github.com/JetBrains/compose-multiplatform-core/pull/557)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,113 @@
...
+
+### Fixes
+- [Fix dynamic framework support](https://github.com/JetBrains/skiko/pull/763)
+- [Fiz TextField context menu](https://github.com/JetBrains/compose-multiplatform/issues/3276)
+- [Fix complex blending on iOS](https://github.com/JetBrains/skiko/pull/728)
+- [ViewConfiguration.touchSlop value is quite low on iOS](https://github.com/JetBrains/compose-multiplatform/issues/3397)
+- [Fix topLeftOffset calculation on iOS in Split View](https://github.com/JetBrains/compose-multiplatform-core/pull/678)
+- [Modifier.draggable onDragStopped not called](https://github.com/JetBrains/compose-multiplatform/issues/3310)
+- [UIKitView. Fix lifetime discrepancy within the composition](https://github.com/JetBrains/compose-multiplatform-core/pull/576/files)
+- [Allow higher than 60hz framerates on CADisplayLink on iPhones](https://github.com/JetBrains/compose-multiplatform-ios-android-template/pull/17)
```
Comment (MatkovIvan) [inline CHANGELOG.md:39]:
```suggestion
- [`ViewConfiguration.touchSlop` value is quite low on iOS](https://github.com/JetBrains/compose-multiplatform/issues/3397)
- [Fix `topLeftOffset` calculation on iOS in Split View](https://github.com/JetBrains/compose-multiplatform-core/pull/678)
- [`Modifier.draggable` `onDragStopped` not called](https://github.com/JetBrains/compose-multiplatform/issues/3310)
- [`UIKitView`. Fix lifetime discrepancy within the composition](https://github.com/JetBrains/compose-multiplatform-core/pull/576/files)
- [Allow higher than 60hz framerates on `CADisplayLink` on iPhones](https://github.com/JetBrains/compose-multiplatform-ios-android-template/pull/17)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,113 @@
...
+
+### API Changes
+ - Resource management was reimplemented. Follow [the guide in the PR](https://github.com/JetBrains/compose-multiplatform/pull/3340) to support new feautures
+
+## Desktop
+
+### Features
+- [Swing interop. Experimental off-screen rendering on graphics](https://github.com/JetBrains/compose-multiplatform-core/pull/601)
+- [Swing interop. ComposePanel that can be disposed manually](https://github.com/JetBrains/compose-multiplatform-core/pull/620)
+- [Add semantic properties to DialogWindow, Popup and Dialog](https://github.com/JetBrains/compose-multiplatform-core/pull/698)
```
Comment (MatkovIvan) [inline CHANGELOG.md:58]:
```suggestion
- [Add semantic properties to `DialogWindow`, `Popup` and `Dialog`](https://github.com/JetBrains/compose-multiplatform-core/pull/698)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,113 @@
...
+- [Use Segoe UI on Windows by default](https://github.com/JetBrains/compose-multiplatform-core/pull/557)
+
+### Fixes
+- [Fix loosing frames on macOs](https://github.com/JetBrains/skiko/pull/753)
+- [Scrolling LazyColumn via mouse wheel stops working](https://github.com/JetBrains/compose-multiplatform/issues/3366)
```
Comment (MatkovIvan) [inline CHANGELOG.md:63]:
```suggestion
- [Scrolling `LazyColumn` via mouse wheel stops working](https://github.com/JetBrains/compose-multiplatform/issues/3366)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,118 @@
...
+- Test framework fixes
+  - [Implement idling resources for tests](https://github.com/JetBrains/compose-multiplatform-core/pull/599)
+  - [Implement `SemanticsNodeInteraction.captureToImage()`](https://github.com/JetBrains/compose-multiplatform-core/pull/589)
+  - [Wait until compose is idle before returning from `DesktopTestOwner.getRoots()`](https://github.com/JetBrains/compose-multiplatform-core/pull/588)
+  - [When the test main clock is advanced, call `ComposeScene.render` with the current test time](https://github.com/JetBrains/compose-multiplatform-core/pull/584)
+  - [Add timestamps to batched test input events, and advance the test clock accordingly when sending them](https://github.com/JetBrains/compose-multiplatform-core/pull/578)
+  - [Tests support semanatic nodes inside `Window` and `DialogWindow`](https://github.com/JetBrains/compose-multiplatform-core/pull/697)
+
+### API Changes
+- [Rename desktop's `Dialog` to `DialogWindow`](https://github.com/JetBrains/compose-multiplatform-core/pull/661)
```
Comment (MatkovIvan) [inline CHANGELOG.md:81]: Let's mention changing default Esc behaviour (https://github.com/JetBrains/compose-multiplatform-core/pull/712)

Comment (igordmn) [reply CHANGELOG.md:81]: I mentioned it for Popup:
```
[All `Popup` overloads call dismiss on `Esc` key by default](https://github.com/JetBrains/compose-multiplatform-core/pull/712)
```
Didn't mention it for `Dialog` (we didn't have it before) and for `DialogWindow` (we don't close it on Esc, I assume)

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,118 @@
...
+### Fixes
+- [Workaround `ComposeWindow` multiple event listeners on resize](https://github.com/JetBrains/compose-multiplatform-core/pull/692)
+- [Fix `Modifier.pointerHoverIcon` for browser apps](https://github.com/JetBrains/compose-multiplatform-core/pull/629)
+- [Fux `RoundedCornerShape` with zero corners](https://github.com/JetBrains/compose-multiplatform/issues/3013)
+- [Fix `ScrollConfig` in web target](https://github.com/JetBrains/compose-multiplatform-core/pull/628)
+- Implemented low-level functions to avoid random crashes:
+  - [Implement cache key in PlatformFont](https://github.com/JetBrains/compose-multiplatform-core/pull/671)
+  - [implement platform detection to apply correct KeyMapping and FontFamiliesMapping](https://github.com/JetBrains/compose-multiplatform-core/pull/637)
+  - [Implement PlatformLocale and StringDelegate for k/wasm](https://github.com/JetBrains/compose-multiplatform-core/pull/625)
+  - [Add missing getAndIncrement in AtomicLong implementation for web](https://github.com/JetBrains/compose-multiplatform-core/pull/570)
```
Comment (MatkovIvan) [inline CHANGELOG.md:98]: Please use ` for code like function and class names

**Review by MatkovIvan — APPROVED** [review]

File: CHANGELOG.md (hunk 9 above)
Comment (igordmn) [reply CHANGELOG.md]: Done

**Pull request merged**